### ISSUE TITLE
[Suggestion][Request][1.12.2] Add aqua plants similar to kelp

### ISSUE THREAD

**Issue by Joccob** [Shadows-of-Fire/Plants#121]
It would be really cool to have some aqua plants that can be harvest and plant in the pond for decorations. Even in the future versions (1.13/1.14), it still seems boring to have kelp as the only plant in the sea and lakes.

**Comment by Shadows-of-Fire** [comment]
Yeah it's something I would like to explore.  I need to majorly redo plants to proceed forward into 1.14, so it may be a bit before any changes happen.
